src/App.css

```diff
@@ -4,18 +4,17 @@
 .App {
   text-align: center;
   display: grid;
-  grid-template-rows: 15vh auto 10vh;
+  grid-template-rows: 10vh auto 10vh;
   height: 100vh;
-  background: #fc0303;
+  background: #0c0d0d;
```

Comment:
Would it be a good idea to use some css variables to get to know what color this is? Another point for a variable would be, that you could easily use it again and don't try to find the right hex again.